ft/ft/command.py

```diff
@@ -30,7 +30,9 @@ def get_argument_parser(self):
         parser.add_argument('args', help='optional arguments', nargs='*')
         parser.add_argument('--column', '-c', type=int,
                             help='apply function to a specific column')
-
+        parser.add_argument('--negate', '-n',
```

Comment:
In this way, you actually add this argument to every command (not just `filter`). I think you could override the `add_command_arguments` function in `filter` and add the additional argument there?